package.json

```diff
@@ -19,6 +19,12 @@
     "prettier-plugin-solidity": "^1.0.0-beta.13",
     "solhint": "^3.3.6",
     "solhint-plugin-prettier": "0.0.5",
-    "solidity-coverage": "^0.7.16"
+    "solidity-coverage": "^0.7.16",
+    "lint-staged": "^13.1.0"
+  },
+  "lint-staged": {
+    "*.sol": "forge fmt",
+    "*.json": "yarn prettier",
+    "*.yml": "yarn prettier"
```

Comment:
I don't think it's worth installing `prettier` for such isolated use cases. I am in favor of dropping the usage of prettier everywhere in this repo